.github/composite_actions/run_xcodebuild_test/action.yml

```diff
@@ -14,5 +14,5 @@ runs:
   steps:
     - name: Test ${{ inputs.scheme }}
       run: |
-        xcodebuild test -scheme AWSCognitoAuthPlugin -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 13' | xcpretty --simple --color --report junit && exit ${PIPESTATUS[0]}        
+        xcodebuild test -scheme ${{ inputs.scheme }} -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 13' | xcpretty --simple --color --report junit && exit ${PIPESTATUS[0]}        
```

Comment:
Should we pipe this through an environment variable?  I know we (people with write access to the repo) should be the only ones able to control the input here, but we're still using it directly in a bash substitution, so I think reading it into an environment variable and then referencing that in the bash line would be safer